3684/1/top_gems.rb

```diff
@@ -0,0 +1,5 @@
+require_relative 'gem_searcher'
+
+searcher = GemSearch.new('gems.yaml')
```

Comment:
there's no file extension `.yaml`

Reply:
do you process `--file` flag?

Reply:
yes, but if file isn't .yaml it go on work )